Subject: [PATCH] export c2p

From: Adrian Bunk <bunk@kernel.org>

This patch fixes the following build error:

<--  snip  -->

...
  Building modules, stage 2.
  MODPOST 1203 modules
ERROR: "c2p" [drivers/video/amifb.ko] undefined!
...
make[2]: *** [__modpost] Error 1

<--  snip  -->

Reported-by: Adrian Bunk <bunk@kernel.org>
Signed-off-by: Adrian Bunk <bunk@kernel.org>
Signed-off-by: Geert Uytterhoeven <geert@linux-m68k.org>
---
 drivers/video/c2p.c |    1 +
 1 file changed, 1 insertion(+)

--- a/drivers/video/c2p.c
+++ b/drivers/video/c2p.c
@@ -227,5 +227,6 @@ void c2p(u8 *dst, const u8 *src, u32 dx,
 	dst += dst_nextline;
     }
 }
+EXPORT_SYMBOL_GPL(c2p);
 
 MODULE_LICENSE("GPL");
